ingestion/create_deploy.sh

```diff
@@ -1,17 +1,25 @@
 #!/bin/bash
 
-echo "RALLY_API_URL: $RALLY_API_URL"
-echo "RALLY_WORKSPACE_OID: $RALLY_WORKSPACE_OID"
-echo "DEPLOY_COMPONENT_NAME: $DEPLOY_COMPONENT_NAME"
-echo "DEPLOY_BUILD_ID: $DEPLOY_BUILD_ID"
-echo "DEPLOY_START_TIME: $DEPLOY_START_TIME"
-echo "DEPLOY_END_TIME: $DEPLOY_END_TIME"
-echo "DEPLOY_IS_SUCCESSFUL: $DEPLOY_IS_SUCCESSFUL"
-echo "DEPLOY_MAIN_REVISION: $DEPLOY_MAIN_REVISION"
-echo "PREVIOUS_SUCCESS_BUILD_COMMIT: $PREVIOUS_SUCCESS_BUILD_COMMIT"
-echo "CURRENT_BUILD_COMMIT: $CURRENT_BUILD_COMMIT"
-echo "GIT_REPO_LOC: $GIT_REPO_LOC"
-
+VERSION=1
+NAME=$(basename $0)
+AUTHOR="Iron Will Beard"
```

Comment:
oh no

Reply:
Maybe the metadata vars should be passed in from the consuming job?

Reply:
I also started thinking like that.  I started thinking we wanted more info about the calling script (jenkins, github, whatever).  But Andrew wanted info about the script (as in create_deploy and update_deploy) in the meta...and the version of them....which I get, esp as far as version goes.   If we change them, and rev them, and we can see that someone is using an old version.  Or if they download them and make modifications, hopefully they'll update the author bits.  I get that.   And then I thought about all the info we get from the consuming job and thought, well that's "plenty" of info about that system.  Maybe could be more...but not exactly see what that would be.